massa-execution-exports/src/types.rs

```diff
@@ -69,6 +69,12 @@ pub struct ReadOnlyExecutionRequest {
     pub call_stack: Vec<ExecutionStackElement>,
     /// Target of the request
     pub target: ReadOnlyExecutionTarget,
+    /// optional event status
+    ///
+    /// Some(true) means final
+    /// Some(false) means candidate
+    /// None means final _and_ candidate
```

Comment:
I don't think it's possible to make the execution for final and candidate, this could be only a boolean.

Reply:
IMHO, we should be more explicit about this parameter. 

```rust
    /// Some(true) means start execution from final state
    /// Some(false) or None means start execution from active state
```

Reply:
If it's only true or false we don't need an option.

Reply:
I totally agree, we don't have to add a third state here. 

done in 0e537fca9c63fdb9101fe9b69f5ece480fc86135 as it's a part 2 of this PR. 

Sorry for miss.